pkg/utils/utils.go

```diff
@@ -13,8 +13,8 @@ import (
 func FletcherChecksum8(inp string) uint8 {
 	var ckA, ckB uint8
 	for i := 0; i < len(inp); i++ {
-		ckA = (ckA + inp[i]) & 0xf
-		ckB = (ckB + ckA) & 0xf
+		ckA = (ckA + inp[i]) % 0xf
+		ckB = (ckB + ckA) % 0xf
```

Comment:
One other option would be to switch just this line back to &, which would allow us to get to a range of 0 to 254, which is exactly what we want. It wouldn't technically be fletcher 8 then, but I'm not sure how much we care about that.